### DISCUSSION TITLE
Recurring Shifts

### DISCUSSION THREAD

**Discussion by ArturoTrailblazer** [Sundae-Shop-Consulting/Volunteers#48] [Feature Design]
What are items should we factor into the recurrence needs.

**Comment by lmeerkatz** [comment]
@ArturoTrailblazer The initial ([very rough sketch](https://github.com/Sundae-Shop-Consulting/Volunteers/wiki/MVP-Features#v4s-volunteers-dashboard)) concept is to use PMM's scheduling model but make it specific to Volunteer Shifts instead of Service Schedules. Knowing that that isn't much to go on, were there specific things you want to make sure we're accounting for?
